intents/intents.py

```diff
@@ -0,0 +1,115 @@
+"""This module currently contains the signature/implementation of some intents"""
+
+import aspects
+
+import datetime
+
+def top_k(table, metric, dimensions, is_asc, k, **kwargs):
+    """This function will implement the top-k intent
+
+    Sorts tuples in the order metric, after applying slice, groupby operations. Also removes the tuples that do not lie in the given date range.
```

Comment:
Let's stick to 80-char limit for a line as per google style guide: http://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
ok, will break larger lined into 2/3 lines.